examples/boundary_comp.py

```diff
@@ -6,7 +6,7 @@
 from atoMEC import models, Atom, config
 
 # use parallelization to make things slightly quicker
-config.numcores = -1
+config.numcores = 5
```

Comment:
How does this change relate to this PR? Why specifying a fix value? What happens if you do not have 5 cores available or they are fully consumed by other processes?

Reply:
You're right, it doesn't particularly relate to this PR. It was just that at the time I updated the examples for this PR, `config.numcores=-1` wasn't actually behaving as it should.

Now that #102 is merged we can either go back to `config.numcores=-1` or comment that part out and leave it to the user. I just wanted to show how one could in principle take advantage of parallelization. Up to you which would be better.

Reply:
I would leave this as is. Because the change from -1 to 5 is not motivated by this PR and value 5 seems to be chosen arbitrarily.

Reply:
yup ok, changed back.

(for the record, there was a reason for the value 5, as the optimal number of cores is usually `config.lmax + 1`, and in this instance `config.lmax=4`. But `joblib` is supposed to work out the optimal number anyway if you specify `-1`, so this is probably a better choice regardless)

Reply:
Ok, but than on would have better specified `config.numcores = config.lmax + 1`, right? Either way, let's stick with the -1.:)

BTW: You could have also made this change without making a new commit, by just editing the `boundary_comp.py` and then doing a `git commit --amend --no-edit` and afterwards a force push. Not sure what would have happened to this thread though.

Reply:
In theory you're right, in practice the way I structured the code was not quite optimal and this wouldn't work unless `config.numcores` was specified _after_ the `ISModel` object is built. An overhaul of the code structure is on my mind (#96). Anyway all is ok with the -1 for now :)

Ah ok, thanks for the tip